MainPanel.c

```diff
@@ -81,7 +82,7 @@ static HandlerResult MainPanel_eventHandler(Panel* super, int ch) {
 
    if (EVENT_IS_HEADER_CLICK(ch)) {
       int x = EVENT_HEADER_CLICK_GET_X(ch);
-      int hx = super->scrollH + x + 1;
+      size_t hx = super->scrollH + (unsigned int)x + 1;
```

Comment:
Shouldn't the second cast also be `size_t`?

Reply:
@BenBE Let me explain a bit:

* `signed int` cast to `size_t`: Perform a sign extension.
* `unsigned int` cast to `size_t`: Perform a zero extension.

When I made this patch, I made a personal rule that, if a zero extension would suffice, I would avoid sign extension. This also applies to the noisy-as-you-called-it changes about me casting the `xSnprintf` results to `size_t`.